src/View3d.ts

```diff
@@ -925,6 +925,20 @@ export class View3d {
     };
     Object.assign(pane.element.style, paneStyle);
 
+    const cacheSettings = {
+      concurrency: 4,
+      size: 1000000,
+    };
+    const caching = pane.addFolder({ title: "Cache" });
+    caching.addInput(cacheSettings, "concurrency").on("change", ({ value }) => {
+      // TODO could/should we treat the queue as more of a global object?
+      this.image?.volume.loader?.setConcurrencyLimit(value);
+    });
+    caching.addInput(cacheSettings, "size").on("change", ({ value }) => {
```

Comment:
do these only change when there is a new image or are there other conditions where these settings would change?

Reply:
Adding them here is just to make them tweakable during testing/debugging to help find optimal settings.   A regular user would not have a reason to change these, typically.
  
IMO the cache size and the max fetch concurrency should be thought of as global values for the whole viewer instance, but for now they are fairly tightly associated with a IVolumeLoader and they really only matter for zarr data.